core/assets/bundles/bundle.properties

```diff
@@ -121,7 +121,7 @@ committingchanges = Committing Changes
 done = Done
 feature.unsupported = Your device does not support this feature.
 
-mods.initfailed = [red]⚠[] The previous Mindustry instance failed to initialize. This was likely caused by misbehaving mods.\n\nTo prevent a crash loop, [red]all mods have been disabled.[]
+mods.initfailed = [red]âš [] The previous Mindustry instance failed to initialize. This was likely caused by misbehaving mods.\n\nTo prevent a crash loop, [red]all mods have been disabled.[]
```

Comment:
what was changed here

Reply:
Not sure, my bundle pr also had changes to random lines with no apparent difference

Reply:
<strike>I dont know</strike>

Reply:
> what was changed here

<strike>I dont really know</strike>